app/api/api/views.py

```diff
@@ -99,6 +101,16 @@
     _save_mapping_rules,
     delete_mapping_rules,
 )
+import random
+import string
+import datetime
+from azure.storage.blob import BlobServiceClient, ContentSettings
+from shared.services.azurequeue import add_message
+
+
+def modify_filename(filename, dt, rand):
```

Comment:
please add the type annotations to this method. it can also go in the services file for now instead of views.